man/org.freedesktop.systemd1.xml

```diff
@@ -1303,13 +1316,18 @@ node /org/freedesktop/systemd1 {
         <listitem><para>The unit object path</para></listitem>
       </itemizedlist></para>
 
-      <para><function>Subscribe()</function> enables most bus signals to be sent out. Clients which are
+      <para><function>SubscribeWithFlags()</function> enables most bus signals to be sent out. Clients which are
       interested in signals need to call this method. Signals are only sent out if at least one client
-      invoked this method. <function>Unsubscribe()</function> reverts the signal subscription that
-      <function>Subscribe()</function> implements. It is not necessary to invoke
-      <function>Unsubscribe()</function> as clients are tracked. Signals are no longer sent out as soon as
-      all clients which previously asked for <function>Subscribe()</function> either closed their connection
-      to the bus or invoked <function>Unsubscribe()</function>.</para>
+      invoked this method. <function>UnsubscribeWithFlags()</function> reverts the signal subscription that
+      <function>SubscribeWithFlags()</function> implements. It is not necessary to invoke
+      <function>UnsubscribeWithFlags()</function> as clients are tracked. Signals are no longer sent out as
+      soon as all clients which previously asked for <function>SubscribeWithFlags()</function> either closed
```

Comment:
Preexisting condition, but not "asked for SubscribeWithFlags()", but just "called SubscribeWithFlags()".